src/box/lua/config/applier/credentials.lua

```diff
@@ -1,4 +1,5 @@
 local log = require('internal.config.utils.log')
```

Comment:
> NO_TEST=yet

AFAIU, it is planned to add. Correct?

I propose to don't go crazy with tests, but have at least one chap-sha1 and one pap-sha256 test, where we actually perform authorization using net.box.

Reply:
> AFAIU, it is planned to add. Correct?

Definitely.

Reply:
Done.